lib/service/trello.rb

```diff
@@ -1,57 +1,62 @@
-require 'rest-client'
-require 'addressable/uri'
+module Service
+  module Trello
+
+    USER_NAME         = 'redwingsruby'
+    LIST_TASKS        = 'tasks'
+    BOARD_PROCESS     = 'PROCESS'
+    BOARD_KNOWLEDGE   = 'KNOWLEDGE'
+    ORGANIZATION_NAME = 'rubyredwings'
+
+    def self.boards_backup
+      TrelloApi::Member.boards(USER_NAME).each do |board|
+        board_backup = TrelloBackup.create!(board: board[:name], data: TrelloApi::Board.data(board[:id]))
+      end
+    end
 
-module Service::Trello
+    def self.setup_user(user)
 
-  TRELLO_USER_NAME = 'redwingsruby'
+      email     = user.email
+      full_name = user.full_name
```

Comment:
User.first.full_name
  User Load (6.0ms)  SELECT  "users".* FROM "users"  ORDER BY "users"."id" ASC LIMIT 1
NoMethodError: undefined method `full_name' for #<User:0x007faf44abc700>

???